fe/fe-core/src/test/java/com/starrocks/sql/plan/OuterJoinReorderTest.java

```diff
@@ -16,13 +16,10 @@
 
 import com.google.common.collect.Lists;
 import com.starrocks.common.FeConstants;
+import org.junit.Test;
 import org.junit.jupiter.api.BeforeAll;
```

Comment:
junit4 test, beforeAll is unused, so the case modify is right?

Reply:
> junit4 test, beforeAll is unused, so the case modify is right?

what do you mean?

Reply:
Why not use the ParameterizedTest? ParameterizedTest could sperate each sql to a individual case. If one case failed, it doesn't affect other case.

Reply:
> Why not use the ParameterizedTest? ParameterizedTest could sperate each sql to a individual case. If one case failed, it doesn't affect other case.

but if fails, it's hard to find the case failed